src/components/InputData.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function InputData() {
+  return (
+    <div>
+      <input type="text" className="input-form" placeholder="Enter Book Name" />
+      <input type="text" className="input-form" placeholder="Enter Author Name" />
+      <input type="button" className="input-form" value="Enter " />
```

Comment:
[Optional] I suggest you make `input` tag `type="submit"` instead of `type="button"` since its a form submission 👍